test/catch2/live_restore/utils_live_restore.cpp

```diff
@@ -0,0 +1,86 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "utils_live_restore.h"
+#include <fstream>
+namespace utils {
+
+/*
+ * Create a string representation of an extent list, for example (1-10),(15-30) represents an extent
+ * list with holes at ranges 1 to 10 and 15 to 30 (inclusive).
+ */
+std::string
+extent_list_str(WT_LIVE_RESTORE_FILE_HANDLE *lr_fh)
+{
+    WT_LIVE_RESTORE_HOLE_NODE *ext = lr_fh->destination.hole_list_head;
+
+    std::string str = "";
+
+    if (ext == NULL) {
```

Comment:
Brackets, and the if two below. Also `nullptr` not `NULL`.

Reply:
[3b7771b](https://github.com/wiredtiger/wiredtiger/pull/11332/commits/3b7771b399d3c9323c6d67572df9faf8a7eb1f5e)